### ISSUE TITLE
reomte gpu usage

### ISSUE THREAD

**Issue by HaiFengZeng** [taichi-dev/taichi#4390]
Well,taichi is amazing if it can work with a gpu,however, I have a two computers and one is a server with a gpu, is there any ideas to use the gpu from this remote accesss? Thank you!

**Comment by k-ye** [comment]
Hi there, you can check out [X11 forwarding](https://www.answertopia.com/ubuntu/displaying-ubuntu-applications-remotely-x11-forwarding/).

**Comment by bobcao3** [comment]
Keep in mind that X11 forwarding only works with old GUI

**Comment by bobcao3** [comment]
we might be able to do GGUI with forwarding is there a demand

**Comment by DongqiShen** [comment]
> Keep in mind that X11 forwarding only works with old GUI

@bobcao3 Hi, I tried X11 forwarding but it doesn't work. My machine is Mac (M1) and remote access to ubuntu 20.04, so what you mean is that taichi doesn't support X11 forwarding officially ? Thanks.

**Comment by k-ye** [comment]
> However, the performance is really terrible

Yeah, that's mostly due to X11/network settings. Taichi can help little in this case :-(

**Comment by shouldsee** [comment]
I stumbled into the slow X11 performance issue as well. It came out possible to 
accelerate the network transfer with a vnc-protocol instead of X11 forwarding. 
I documented my setup here. https://github.com/shouldsee/taichi_vnc_docker
